docs/en/integrations/data-ingestion/clickpipes/secure-kinesis.md

```diff
@@ -0,0 +1,84 @@
+---
+slug: /en/integrations/clickpipes/secure-kinesis
+sidebar_label: Kinesis Role-based Access
+title: Kinesis Role-based Access
+---
+
+This article demonstrates how ClickPipes customers can leverage role-based access to authenticate with Amazon Kinesis and access their data streams securely.
+
+## Introduction
+
+Before diving into the setup for secure Kinesis access, it's important to understand the mechanism. Here's an overview of how ClickPipes can access Amazon Kinesis streams by assuming a role within customers' AWS accounts.
+
+![securekinesis](@site/docs/en/integrations/data-ingestion/clickpipes/images/securekinesis.jpg)
```

Comment:
What is CBD stream?

Reply:
Whoops, good catch, that should be CDC (Change data capture)